examples/loki/nurf_data.py

```diff
@@ -102,120 +102,205 @@ def nurf_file_creator(loki_file, path_to_loki_file, data):
 
     # open the file and append
     with h5py.File(loki_file, 'a') as hf:
+                
+        #comment on names
+        #UV/FL_Background is the dark
+        #UV/FL_Intensity0 is the reference
+        #UV/FL_Spectra is the sample
         
+        # image_key: number of frames (nFrames) given indirectly as part of the shape of the arrays 
+        # TODO: keep in mind what happens if multiple dark or reference frames are taken
+        
+        # remove axis=2 of length one from this array
+        data['UV_spectra']=np.squeeze(data['UV_spectra'], axis=2)  #this removes the third axis in this array, TODO: needs later to be verified with real data from hardware
+        
+        # assemble all spectra in one variable
+        uv_all_data=np.row_stack((data['UV_spectra'],data['UV_background'], data['UV_intensity0']))
+      
+        
+        # assemble image_key 
+        # #TODO: needs later to be verified with real data from hardware
+        uv_nb_spectra=np.shape(data['UV_spectra'])[0]
+        uv_ik_spectra=np.zeros((1,uv_nb_spectra))  #interperation here: 0 for sample (in comparison to projections)
```

Comment:
Why is there an additional dimension of length 1 in the key? It should just be 1d, shouldn't it?

Reply:
That is true. Well spotted. I don't need this. Done.